tests/setup-env.js

```diff
@@ -1 +1,2 @@
 import '@testing-library/jest-dom/extend-expect'
+jest.mock('scheduler', () => require('scheduler/unstable_mock'))
```

Comment:
Biggest unknown for to figure out. I think we need to use this to advance time (or something like that) if we have tests scheduling work (like setState).